dea/apps/index_from_tar.py

```diff
@@ -51,7 +52,8 @@ def cli(input_fname,
         verify_lineage,
         ignore_lineage,
         gzip,
-        xz):
+        xz,
+        protocol):
 
```

Comment:
I think we should also accept just `s3` or `gs` for `--protocol` option.

```python
if not protocol.endswith('://'):
    protocol = protocol + '://'
```

Reply:
rasterio currently supports the following schems:

```python
SCHEMES = {
    'ftp': 'curl',
    'gzip': 'gzip',
    'http': 'curl',
    'https': 'curl',
    's3': 's3',
    'tar': 'tar',
    'zip': 'zip',
    'file': 'file',
    'oss': 'oss',
    'gs': 'gs',
}
```

Reply:
Done